docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -25,7 +25,7 @@ which you'll need to enable mutual TLS on your MySQL server.
 ## Configure MySQL Server
 
 To configure MySQL server to accept TLS connections, add the following to
-MySQL configuration file `mysql.cnf`:
+a MySQL custom configuration file such as `/etc/mysql/conf.d/teleportmysql.cnf`:
```

Comment:
@stevenGravy Have a couple of questions regarding this:
- Is this valid for both older pre-8.0 MySQL and more recent 8.0+ versions?
- Do you know if this directory is valid for all Linux distros? Also, maybe add a note for users configuring MySQL on macOS/Win machines as well?